modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -90,16 +90,16 @@ This command also supports referencing an installer package already downloaded o
 +
 The command outputs a confirmation indicating the upgrade is being performed in the background.
 +
-. Run `gravity status` on a node and verify that the cluster status is “updating”.
+. Run `sudo gravity status` on a node and verify that the cluster status is “updating”.
 . Follow the progress on the upgrade procedure:
 .. Using your terminal open on a controller node, run the following command: 
 +
 ----
-sudo ./gravity plan
+sudo gravity plan
 ----
 +
 . Confirm the upgrade has completed successfully:
-.. Run `sudo ./gravity status` and verify that the cluster status transitioned from “updating” to “active”.
+.. Run `sudo gravity status` and verify that the cluster status transitioned from “updating” to “active”.
```

Comment:
could we verify this is okay in the upgrade scenario, to use the gravity binary in the PATH vs the one that's in the local upgrade directory?
@jeff-1amstudios

Reply:
in this case, the PATH one because we want to assess the state of the current cluster